CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Unreleased
 
 - Added some badges to the README
+- Marked `AddRazorEmailRenderer` obsolete, use `AddRazorMailRenderer` instead
```

Comment:
Hmm, I agree with the first change, but the other two I'm leaning towards being too ambiguous?

Reply:
And isn't that contradictory with the change of renaming `MailRenderer` to `RazorMailRenderer`?

Reply:
In my mind I read it like `Add<Service>`, so when I want to add and implementation of `IMailRenderer` and that implementation is `RazorMailRenderer` I look for an extension method `AddRazorMailRenderer`.

(_maaaaybe_ in the future we add another implementation of `IMailRenderer` and that should get a different extension method)

Reply:
Right. I guess my worry here is that `IMailRenderer` is too broad a name for our library to use.